http/src/main/resources/reference.conf

```diff
@@ -258,6 +259,22 @@ azure {
          path = "/"
        }
      ]
+     enabled = true
+   }
+
+   hail-batch-app-config {
+     chart-name = "/leonardo/hail-batch-terra-azure"
+     chart-version = "0.1.4"
+     release-name-suffix = "hail-rls"
+     namespace-name-suffix = "hail-ns"
+     ksa-name = "hail-ksa"
+     services = [
+       {
+         name = "batch"
+         kind = "ClusterIP"
+       }
+     ]
+     enabled = true
```

Comment:
I introduced an `enabled` flag for all app types (defaulting to true). Was just thinking we could potentially merge and disable hail-batch in prod until compliance review happens..